scripts/graph_accelerometer.py

```diff
@@ -56,7 +57,7 @@ def calc_specgram(data, axis):
     M = 1 << int(.5 * Fs - 1).bit_length()
     window = np.kaiser(M, 6.)
     def _specgram(x):
-        return matplotlib.mlab.specgram(
+        return mlab.specgram(
```

Comment:
FWIW, I'd avoid this change as matplotlib has "funky" order of import rules wrt calling `matplotlib.use('Agg')`.  It's easier to use the longer mlab name explicitly.

-Kevin